### PR TITLE
Pranay Migrate client and common network enpoints

### PR DESCRIPTION
**Pull request by ptulugu**

### CODE REVIEW

File: rita_client/src/rita_loop/mod.rs
Comment (jkilpatr) [inline rita_client/src/rita_loop/mod.rs:103]: I don't see any migration to AWC for heartbeats here, so I don't believe this will work. Heartbeats will simply not be sent.

File: rita_common/src/network_endpoints/mod.rs
Comment (jkilpatr) [inline rita_common/src/network_endpoints/mod.rs:21]: are we sure we still need this? I think this sort of conversion is done automatically by actix web now.

File: rita_common/src/network_endpoints/mod.rs
Comment (jkilpatr) [inline rita_common/src/network_endpoints/mod.rs:68]: keeping the result return type here is important. What about the error handling presented a problem with keeping it that way?

**Review by jkilpatr — CHANGES_REQUESTED** [review]
Looks good overall. My biggest statement is that the commit message should indicate that LightClientManager was migrated and generally include more details about what was done.

In general keeping the return type of the endpoints as a Result<Json<whatever>> is by far the best. You can use .responder() to perform a converstion to http response that will be more accurate than trying to do it yourself. 

The big issue remaining is that the send_udp_heartbeat will not work the way it's been modified here. An old actix context is required for it to send it's dns requests. This change may have to wait on heartbeats being migrated before it can be merged.

File: rita_common/src/network_endpoints/mod.rs
Comment (ptulugu) [reply rita_common/src/network_endpoints/mod.rs:21]: fixed

**Pull request merged**
